pkg/node/node.go

```diff
@@ -570,13 +570,13 @@ func addTaint(node *corev1.Node, nth Node, taintKey string, taintValue string, e
 		}
 
 		if err != nil {
-			log.Log().
+			log.Warn().
 				Str("taint_key", taintKey).
 				Str("node_name", node.Name).
 				Msg("Error while adding taint on node")
 			return err
 		}
-		log.Log().
+		log.Warn().
```

Comment:
Info log?

Reply:
I think this one should be warn since it's reporting a non critical error

Reply:
nvm, you're right,  this is critical